### PR TITLE
Fix separator bugs

### PR DESCRIPTION
**Pull request by pantor**
This PR should fix the localization of the separator (#49) as well as truncated results (#48).

### CODE REVIEW

File: test/CoreTest.vala [hunk 1]
```diff
@@ -23,6 +23,7 @@ class PantheonCalculator.Core.CoreTest : Object {
         assert_equal ("0+0", "0");
         assert_equal ("2+2", "4");
         assert_equal ("4.23 + 1.11", "5.34");
+        assert_equal ("25.123 - 234.2", "-209.077");
```
Comment (danrabbit) [inline test/CoreTest.vala]: Can you add comments for the relevant issue reports please :)

**Comment by pantor** [comment]
Oh, I've found an error...

**Comment by pantor** [comment]
I'm sorry for closing and reopening this PR, but the bug fix was not complete... I've now added tests for the localization, which however needed a little bit of restructuring the code. I hope it all works better now!

**Comment by pantor** [comment]
There was indeed another issue with my code, which was sadly not catched by the unit tests. I've now tested it with pl_PL and it works on my machine.

**Comment by pantor** [comment]
This PR now also fixes issue #52, so I closed PR #55.

**Pull request merged**